contracts/LifInterface.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.4.11;
+
+import 'zeppelin-solidity/contracts/token/ERC20.sol';
+
+/**
+ * @title Lif token interface
+ */
+contract LifInterface is ERC20 {
+  uint public maxSupply;
```

Comment:
To see the difference about the total tokens ins distributions and the maximun supply, in maxSupply we take in count the future payments for example, and the users would be able to see how much of the total tokens where distributed. They would be able to see how after the crowdsale during 4 years we reach the 100% token distribution between token buyers and founders.
I think its useful to have the maxSupply determined and written in the contract. So the traders, users, developers can access it easily. And also calculate the % distributed of tokens.